src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -334,6 +338,26 @@ tsi_result alts_tsi_handshaker_result_create(grpc_gcp_HandshakerResp* resp,
                                                  local_service_account);
   grpc_gcp_AltsContext_set_peer_rpc_versions(
       context, const_cast<grpc_gcp_RpcProtocolVersions*>(peer_rpc_version));
+
+// START OF PLUMBING
+  // grpc_gcp_HandshakerResult* ncresult = const_cast<grpc_gcp_HandshakerResult*>(hresult);
+  grpc_gcp_Identity* peer_identity = const_cast<grpc_gcp_Identity*>(identity);
+  if(peer_identity == nullptr ) {
```

Comment:
nit: remove the empty space after "nullptr". Ditto elsewhere.

Reply:
will do!